### PR TITLE
Write a clean script that performs fusion

### PR DESCRIPTION
**Pull request by x65han**
- [Issue 1161](https://github.com/castorini/anserini/issues/1161)
- Supports RRF & combos sum for now.

### CODE REVIEW

File: eval/fusion.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+from typing import List
+from trectools import fusion, TrecRun
+
+
+class FusionMethod(enum.Enum):
+    RRF = 'RRF'
+    COMBO_SUM = 'COMBO_SUM'
+
+
+def gen_trec_run_list(paths: List[str]) -> List[TrecRun]:
```
Comment (lintool) [inline eval/fusion.py:26]: rename `load_trec_runs`?

File: eval/fusion.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+'''
```
Comment (lintool) [inline eval/fusion.py:2]: The boilerplate here got mangled somewhere (other script too)? Use this one: https://github.com/castorini/anserini/blob/master/src/main/python/run_solr_regression.py#L2

File: eval/fusion.py (hunk 1 above)
Comment (x65han) [reply eval/fusion.py]: Addressed

File: eval/fusion.py (hunk 2 above)
Comment (x65han) [reply eval/fusion.py:2]: Copy/paste messed up for some reason. Addressed

File: eval/fusion.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 '''
+
```
Comment (lintool) [inline eval/fusion.py:14]: hey @x65han I don't think you actually fixed this issue?

**Comment by lintool** [comment]
@x65han license boilerplate is still mangled?

**Comment by x65han** [comment]
> @x65han license boilerplate is still mangled?

Fixed attempt 2. Hopefully, it is correct now.

**Pull request merged**